baekjoon/fibonacci.js

```diff
@@ -0,0 +1,12 @@
+let input = require('fs').readFileSync('/dev/stdin').toString();
+const value = Number(input);
```

Comment:
백준 푸실 때 이렇게 한 줄로도 입력 받을 수 있어요! 제가 애용하는 방법이라서 공유해요 ㅎㅎ
```suggestion
const value = parseInt(require("fs").readFileSync("/dev/stdin").toString());
```